colelandolt/personal-website
# Improvements to User Interface

## Description

- Adds personal logo to the navbar
- Adds success message to subscribers
- Adds CSS styling
- Adds heroes to homepage

## Validation
- `python manage.py runserver`
- Submit new entry in the form on the home page
- Confirm that the success message appears and the form disappears
